fake-ipa/README.md

```diff
@@ -0,0 +1,48 @@
+# Fake ironic python agent
+
+Fake ipa is a tool to help test ironic communication with IPA.
```

Comment:
Let's use some consistent spelling of IPA.

Reply:
Maybe "productize" it even a bit and consistently call it `FakeIPA` ?